tasks-by-users/samar4tech/01-e-commerce-website/account.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
```

Comment:
can you explain why we use `DOCTYPE` ?

Reply:
It's an instruction to the browser about what version of HTML, basically the page is written in.